ark/utils/notebooks_test_utils.py

```diff
@@ -732,19 +732,19 @@ def flowsom_cell_visualize(tb, flowsom_dir, fovs,
     tb.inject(cell_overlay_fovs, after='cell_overlay_fovs')
 
     # generate the cell cluster masks
-    tb.execute_cell('cell_mask_gen')
+    tb.execute_cell('cell_mask_gen_save')
 
     # test the saving of cell masks
     # NOTE: no point testing save_cell_masks = False since that doesn't run anything
-    cell_mask_save = """
-        data_utils.save_fov_images(
-            cell_fovs,
-            os.path.join(base_dir, cell_output_dir),
-            cell_cluster_masks,
-            name_suffix='_cell_mask'
-        )
-    """
-    tb.inject(cell_mask_save, 'cell_mask_save')
+    # cell_mask_save = """
+    #     data_utils.save_fov_images(
+    #         cell_fovs,
+    #         os.path.join(base_dir, cell_output_dir),
```

Comment:
And here?

Reply:
To remove it, since we don't check if the user wants to save with the batching system.